iosxr_iso2vbox.py

```diff
@@ -386,6 +378,19 @@ def verboseprint(*args):
         run('VBoxManage export %s --output %s' % (vmname, ova_out))
         verboseprint('Created OVA %s' % ova_out)
 
+    # Run basic sanity tests
+    verboseprint('Testing Virtualbox')
+    iosxr_test_path = os.path.join(pathname, 'iosxr_test.py')
+    cmdstring = "python %s %s" % (iosxr_test_path, box_out)
+    verboseprint("Running: '%s'" % cmdstring)
+    result = (subprocess.check_output(cmdstring, shell=True))
+    if result is False:
+        # Fail noisily
+        print('Failed basic test, box %s is not sane' % box_out)
+        sys.exit(1)
```

Comment:
shorter:

```python
sys.exit('Failed basic test, box %s is not sane' % box_out)
```

Also has the advantage that it will echo to stderr instead of stdout.